src/index.css

```diff
@@ -1,22 +1,13 @@
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
+:root {
+  font-family: var(--n-font-family);
+  color: var(--n-color-text);
+  background-color: var(--n-color-surface);
 }
 
-html,
 body {
-  block-size: 100vh;
   margin: 0;
-}
-
-body {
-  font-family: var(--n-font-family);
-  background-color: var(--n-color-surface-raised);
-}
-
-#root {
-  block-size: 100%;
+  display: flex;
+  place-items: center;
+  min-width: 320px;
+  min-height: 100vh;
```

Comment:
Could we use logical properties instead in this boilerplate to promote that approach?

Reply:
Good idea, will do.

Reply:
Converted to logical properties